index.bs

```diff
@@ -2282,7 +2324,8 @@ when invoked, must run these steps:
             1. Set |request|'s [=request/error=] to |result|.
             1. Set |request|'s [=request/done flag=].
             1. [=Fire an event=] named <a event>`error`</a> at |request| with its
-                {{Event/bubbles}} and {{Event/cancelable}} attributes initialized to true.
+                {{Event/bubbles}} and {{Event/cancelable}} attributes
```

Comment:
Why did we add this line break?

Reply:
No idea. Might have been my text editor.